pkg/test/framework/features/features.yaml

```diff
@@ -29,6 +29,8 @@ features:
       status:
   # features that allow users to secure their services and service mesh.
   security:
+    mtls:
+      k8s-ca:
```

Comment:
Can we put a comment about what this actually means?

This feature is actually not related at all to mtls, its used for 1 way TLS to istiod (currently)?

Reply:
I have moved it to control-plane.